lib/login/googleSignIn.dart

```diff
@@ -3,27 +3,22 @@ import 'package:flutter/material.dart';
 import 'package:google_sign_in/google_sign_in.dart';
 
 class UserAuthentication {
-  UserAuthentication({
-    @required this.googleSignIn,
-    @required this.firebaseAuth,
-  });
+  GoogleSignIn googleSignIn = GoogleSignIn();
```

Comment:
curious as to why you changed the old code to this?

Reply:
It's a bit more clean this way. Found out that you can instantiate these here and you don't have to instantiate again in an init state of another file. Previously we had to instantiate it in login.dart and also settings.dart but now it's only here.

Reply:
sounds good